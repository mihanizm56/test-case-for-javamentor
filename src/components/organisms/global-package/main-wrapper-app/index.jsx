import React, { memo } from "react";
import { MainLayout } from "../../../../routes";
import { Header } from "../../..";
import { AuthStoreProvider, AuthFormProvider } from "../../../../containers";
import "./MainWrapper.css";

export const MainWrapper = memo(props => {
	return (
		<div className="main-wrapper">
			<Header />
			<AuthStoreProvider>
				<AuthFormProvider>
					<MainLayout {...props} />
				</AuthFormProvider>
			</AuthStoreProvider>
		</div>
	);
});
