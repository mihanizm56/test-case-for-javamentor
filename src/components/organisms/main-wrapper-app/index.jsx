import React, { memo } from "react";
import { Searcher } from "../../../modules/searcher";
import "./App.css";

export const MainWrapper = memo(() => {
  return (
    <div className="global-wrapper">
      <div className="main-wrapper">
        <Searcher />
      </div>
    </div>
  );
});
