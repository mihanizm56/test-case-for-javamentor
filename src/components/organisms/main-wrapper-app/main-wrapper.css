.global-wrapper {
    position: relative;
    display: flex;
    flex-direction: column;
    align-items: center;
    justify-content: flex-start;
    width: 100%;
    height: 100vh;
    box-sizing: border-box;
    overflow: hidden;
    border: 1px solid red;
}

.main-wrapper {
    width: 100%;
    height: 100vh;
    display: flex;
    flex-direction: column;
    padding-top: 40px;
    border: 1px solid black;
}