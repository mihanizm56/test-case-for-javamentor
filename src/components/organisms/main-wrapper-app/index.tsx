import React, { memo } from "react";
import { Searcher } from "../../../modules/searcher";
import "./main-wrapper.css";

export const MainWrapper = memo(() => (
    <div className="global-wrapper">
      <div className="main-wrapper">
        <Searcher />
      </div>
    </div>
  ));
