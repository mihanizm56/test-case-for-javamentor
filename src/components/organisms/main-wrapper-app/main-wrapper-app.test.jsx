import React from "react";
import { shallow } from "enzyme";
import { MainWrapper } from "../main-wrapper-app";

describe("test MainWrapper", () => {
  const component = shallow(<MainWrapper />);

  describe("test render contract", () => {
    it("renders Searcher", () => {
      expect(component).toMatchSnapshot();
    });
  });
});
