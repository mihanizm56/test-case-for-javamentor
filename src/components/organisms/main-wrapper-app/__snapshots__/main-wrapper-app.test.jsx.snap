// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test MainWrapper test render contract renders Searcher 1`] = `
<div
  className="global-wrapper"
>
  <div
    className="main-wrapper"
  >
    <Memo() />
  </div>
</div>
`;
